.travis.yml

```diff
@@ -36,7 +36,8 @@ jobs:
       name: "Tests"
       script:
         - npm run lint
-        - npm run test:ember
+        - npx percy exec -- npm run test:ember
+      env: PERCY_TOKEN=ce5e8d59a35851ea07032b21668a6718e8daf4ac8da2f4f3fdcafc8c89dd0a0f
```

Comment:
is this supposed to be public?

Reply:
Yes this is expected to be public, otherwise we wouldn't have Percy diffs from contributors that are submitting PRs from forks of the project.

I am in an open discussion with Percy about this issue because it has been something that needs to be improved for a few years now 🙈